daemon/config/config_linux.go

```diff
@@ -79,6 +80,9 @@ type Config struct {
 	// ResolvConf is the path to the configuration of the host resolver
 	ResolvConf string `json:"resolv-conf,omitempty"`
 	Rootless   bool   `json:"rootless,omitempty"`
+
+	// CDISpecDirs is a list of directories in which CDI specifications can be found.
+	CDISpecDirs []string `json:"cdi-spec-dirs,omitempty"`
```

Comment:
Yes, I noticed the Winows builds failing because of this and I have moved it.

One question about setting the defaults though. Are the defaults not still platform specific at this stage? That is to say, should we use `[]string{"/etc/cdi", "/var/run/cdi"}` on Windows as well as linux, or should the Windows values not just be `[]string{}`?

Reply:
The default _values_ may need to be platform specific, but the code in Moby which takes a copy of the defaults does not necessarily need to be. Package `cdi` could export different values for `cdi.DefaultSpecDirs` when compiled for Windows vs. Linux, for instance.

Reply:
@klihub a quick question: What is the expected behaviour when a CDI `Cache` is configured with an empty slice (or `nil`) as the CDI directories? Does this effectively disable CDI?

Reply:
> The default _values_ may need to be platform specific, but the code in Moby which takes a copy of the defaults does not necessarily need to be. Package `cdi` could export different values for `cdi.DefaultSpecDirs` when compiled for Windows vs. Linux, for instance.

Sure. Given that the values in `cdi.DefaultSpecDirs` are not currently platform specific (set to the same values for all plaforms), would you prefer we pull this in irrespective of platform on the `moby` side, or set moby-local platform-specific defaults by only setting these in `config_linux.go` (`setPlatformDefaults`)?

I have created https://github.com/container-orchestrated-devices/container-device-interface/issues/133 to track updating this on the CDI side of things, but don't expect that to be resolved by the time we wish to merge this.

Reply:
I would prefer it be pulled in irrespective of platform as it's one less thing to have to remember to update on the Moby side to enable Windows CDI support after https://github.com/container-orchestrated-devices/container-device-interface/issues/133 has been resolved.